sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -280,6 +286,9 @@ public void onCreate(SQLiteDatabase db, ConnectionSource connectionSource) {
 			TableUtils.createTable(connectionSource, Outbreak.class);
 			TableUtils.createTable(connectionSource, DiseaseClassificationCriteria.class);
 			TableUtils.createTable(connectionSource, SormasToSormasOriginInfo.class);
+			TableUtils.createTable(connectionSource, Campaign.class);
```

Comment:
There are a few methods in this class where you forgot to add the new classes; please check them again (clearTables and at least one other method)